v22.1/aws-dms.md

```diff
@@ -1,11 +1,11 @@
 ---
-title: Migrate CockroachDB Schemas with AWS Database Migration Service (DMS)
+title: Migrate CockroachDB Data with AWS Database Migration Service (DMS)
```

Comment:
I'm still a little unsure about this. We're not migrated cockroachdb data. We're migrating data to cockroachdb.

What about just `Migrate with AWS Database Migration Service (DMS)`?

Reply:
I like that better. Thanks for the suggestion!